bescort.lic

```diff
@@ -716,6 +716,8 @@ class Bescort
   end
 
   def segoltha(mode)
+    EquipmentManager.new.empty_hands
+    EquipmentManager.new.wear_equipment_set?('swimming')
```

Comment:
What does this do if there is no `swimming` set defined? I assume it either would remove all gear or change nothing?

Reply:
Yes.

Reply:
Sorry, I'm not sure I understand. Which thing does it do if that set is not defined?

Reply:
The Yes was in response to your original inquiry of if the 3rd addition was supposed to be:
'EquipmentManager.new.wear_equipment_set?('standard')'

If there is no 'swimming' gearset defined, it does this:
`echo("Could not find gear set #{set_name}")`